From f8dda44d73795d620c3189303c531dee61b15ce4 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 17 Jun 2014 15:27:46 +0800
Subject: [PATCH] ARM: enable interactive as default cpufreq governor in
 imx_v7_defconfig

enable performance governor;
enable interactive as default cpufreq governor.

Acked-by: Shawn Guo <shawn.guo@linaro.org>
Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 897e566..a6ce418 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -38,6 +38,8 @@ CONFIG_HIGHMEM=y
 CONFIG_CMA=y
 CONFIG_CMDLINE="noinitrd console=ttymxc0,115200"
 CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
+CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
-- 
1.8.0

